### PR TITLE
Bug 7189 - Provide page title in website information dialog

### PR DESCRIPTION
**Pull request by darkwing**
I took a few minutes to mock this one out.  Might need eyes from UX, but if undesired, please feel free to close.

WebsiteTitle

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by darkwing** [comment]
@brampitoyo @AmyYLee Thoughts on adding the title in bold?

**Comment by AmyYLee** [comment]
@darkwing the bold works for me. Also spoke to @brampitoyo and he's good with it too. Thanks!

File: app/src/main/res/layout/quicksettings_website_info.xml [hunk 1]
```diff
@@ -21,7 +33,7 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:text="https://wikipedia.org" />
+        tools:text="" />
```
Comment (NotWoods) [inline app/src/main/res/layout/quicksettings_website_info.xml]: nit: leave in `tools:text` for the Preview page

File: app/src/main/res/layout/quicksettings_website_info.xml [hunk 2]
```diff
@@ -12,6 +12,18 @@
...
+        android:id="@+id/title"
+        style="@style/QuickSettingsText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="8dp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="" />
```
Comment (NotWoods) [inline app/src/main/res/layout/quicksettings_website_info.xml]: nit: For Preview pane,
```suggestion
        tools:text="Wikipedia" />
```

File: app/src/test/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsFragmentStoreTest.kt [hunk 3]
```diff
@@ -71,9 +71,10 @@ class QuickSettingsFragmentStoreTest {
...
     fun `createWebsiteInfoState constructs a WebsiteInfoState with the right values for a secure connection`() {
         val websiteUrl = "https://host.com/page1"
+        val websiteTitle = "Hello"
         val securedStatus = true
 
-        val state = QuickSettingsFragmentStore.createWebsiteInfoState(websiteUrl, securedStatus)
+        val state = QuickSettingsFragmentStore.createWebsiteInfoState(websiteUrl, websiteTitle, securedStatus)
 
         assertAll {
             assertThat(state).isNotNull()
```
Comment (NotWoods) [inline app/src/test/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsFragmentStoreTest.kt]: Can you add assertions for the title to the tests here?

File: app/src/main/res/layout/quicksettings_website_info.xml [hunk 4]
```diff
@@ -12,6 +12,18 @@
...
     android:orientation="vertical">
 
+    <TextView
+        android:id="@+id/title"
+        style="@style/QuickSettingsText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="8dp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
```
Comment (NotWoods) [inline app/src/main/res/layout/quicksettings_website_info.xml]: I think this layout got switch from a `ConstraintLayout` to a `LinearLayout` at some point. All the `app:layout_contraint...` attributes are useless in a `LinearLayout` and should be removed.

**Comment by darkwing** [comment]
I just saw these nits since they came in while I was on PTO -- sorry @NotWoods !  This is ready to view again.

**Pull request merged**